src/routes/crud.py

```diff
@@ -0,0 +1,134 @@
+from datetime import datetime, timedelta
+import select
+from fastapi import HTTPException, Response
+from sqlalchemy.ext.asyncio import AsyncSession
+from database.models import MovieModel as Movie
+from schemas.movies import MovieCreate, MovieUpdate
+
+
+async def get_movie(movie_id: int, db: AsyncSession):
+    res = db.execute(select(Movie).where(Movie.id == movie_id))
+    movie = res.scalar_one_or_none()
+    if movie is None:
+        raise HTTPException(
+            status_code=404, detail="Movie with the given ID was not found."
+        )
+    return movie
+
+
+async def get_movies(db: AsyncSession):
+    res = db.execute(select(Movie))
```

Comment:
It's important to await the result of `db.execute` to ensure the asynchronous operation completes before proceeding. Missing `await` can lead to unexpected behavior.

```suggestion
res = await db.execute(select(Movie))
```